feat!: Alt prop is required for Image (#1739)
Amsterdam/design-system: packages/css/src/components/image/README.md

## Guidelines

> - Always add a description of the image in the `alt` attribute.

Reviewer: I like how [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt#usage_notes) states that “[an] image's alternate text should be able to replace the image without altering the meaning of the page”

## Guidelines

- The `alt` attribute is required, but can be empty.

>   This helps to consider providing an alternate text for every image.

Reviewer 2: ```suggestion
```

- A description is unnecessary for decorative images; use `alt=""` for these.
  Non-visual browsers will then hide the image from the user.
  Examples are images that add little to the nearby text, and pictures that purely contribute to the design or atmosphere of the page (source: [W3C Web Accessibility Initiative](https://www.w3.org/WAI/tutorials/images/decorative/)).
- Do describe the content of the image if the image isn’t only decorative.

>   When choosing alt strings for your images, imagine what you would say when reading the page to someone over the phone without mentioning that there's an image on the page (source: [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt)).

Reviewer 2: ```suggestion
  When choosing a description for your images, imagine what you would say when reading the page to someone over the phone without mentioning that there's an image on the page (source: [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt)).
```

- Every image’s alternate text should be able to replace the image without altering the meaning of the page.

>   You should never use alt for text that could be construed as a caption or title (source: [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt)).

Reviewer 2: ```suggestion
  You should never use `alt` for text that could be construed as a caption or title (source: [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt)).
```
